packages/run-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -107,7 +108,10 @@ export const makeVaultManager = (
   // definition of reschedulePriceCheck, which refers to sortedVaultKits
   // XXX misleading mutability and confusing flow control; could be refactored with a listener
   /** @type {ReturnType<typeof makePrioritizedVaults>=} */
-  let prioritizedVaults;
+  let illiquidVaults;
```

Comment:
I think these are the same `prioritizedVaults` as before. 'illiquid' sounds to me like they have insufficient capitalization and should be liquidated.  Would you mind renaming it back to prioritizedVaults, or something representing them as loans in good standing?

Reply:
Yeah, it's a poor name choice. I was looking for something that described _how_ they're prioritized because `vaultsToLiquidate` are also ordered. But I'll leave that for https://github.com/Agoric/agoric-sdk/issues/4415